client/src/components/ModalComponent.js

```diff
@@ -14,7 +14,9 @@ const ModalComponent = (props) => {
 
   return (
     <div>
-      <Button color="light" onClick={toggle}>{buttonLabel}</Button>
+      <strong>
+      <a href="#" className="nav-link active" onClick={toggle}>{buttonLabel}</a>
```

Comment:
just double-checking that nav-link-active is the right class to use here

Reply:
Yes, it's the right class.